packages/sdk-cli/src/commands/screenshot.ts

```diff
@@ -3,6 +3,7 @@ import * as path from 'path';
 import dateformat from 'dateformat';
 import untildify from 'untildify';
 import vorpal from 'vorpal';
+import { default as openScreenshot } from 'open';
```

Comment:
I think it's better to import this as `open`, as it does nothing specific to screenshots.

Reply:
This allowed me to avoid variable shadowing. (See line 20)

Reply:
Thanks